js-extensions/EXTENSIONS.md

```diff
@@ -12,8 +12,8 @@ JSe `@jenkins-cd/js-extensions` module exports 2 things:
 
 The `ExtensionStore` API is very simple, all public methods are asynchronous:
 
-- `getExtensions(extensionPointName, [type,] onload)`
-    This method will async load data and type information as needed, and call the onload handler with a list of extension exports, e.g. the React classes or otherwise exported references.
+- `getExtensions(extensionPointName, [filter,] onload)`
+    This method will async load data and filter the information as needed, and call the onload handler with a list of extension exports, e.g. the React classes or otherwise exported references.
```

Comment:
If the filtering is specific to the component `dataType` field, then I would prefer that to be explicit in this API i.e. `getExtensions(extensionPointName, [dataTypes,] onload)`.